File: Languages/Python/CashDispenser.py
class CashDispenser: 
    INITIAL_COUNT = 2000 


    def __init__(self): 
        self.count = self.INITIAL_COUNT 

    def dispenseCash(self, amount): 
        billsRequired = amount // 20 
        self.count -= billsRequired 

    def isSufficientCashAvailable(self, amount):
        billsRequired = amount // 20 
        if self.count >= billsRequired: 
            return True 
        else: 
            return False 

File: Languages/Golang/CashDispenser.go
package main

const initialCount = 2000

type CashDispenser struct {
	count int
}

func NewCashDispenser() *CashDispenser {
	return &CashDispenser{
		count: initialCount,
	}

}

func (c *CashDispenser) DispenseCash(amount int) {
	billsRequired := amount / 20
	c.count -= billsRequired
}

func (c *CashDispenser) IsSufficientCashAvailable(amount int) bool {
	billsRequired := amount / 20
	if c.count >= billsRequired {
		return true
	}
	return false
}
